Dockerfiles/Dockerfile-tools

```diff
@@ -32,7 +32,13 @@ RUN set -eux \
 	&& chmod +x /usr/bin/yq \
 	&& yq --version
 
-
+# Download Mozilla Sops (Secrets Management)
+RUN set -eux \
+        && SOPS="$( curl -L -sS -o /dev/null -w %{url_effective} https://github.com/mozilla/sops/releases/latest | sed 's/^.*\///g' )" \
+        && curl -L -sS "https://github.com/mozilla/sops/releases/download/${SOPS}/sops-${SOPS}.linux.amd64" > /usr/bin/sops \
```

Comment:
This now requires a condition to either pull `arm64` or `amd64`. See how it is done with `jq` a few lines above: https://github.com/cytopia/docker-ansible/blob/master/Dockerfiles/Dockerfile-tools#L31